.jshintrc

```diff
@@ -16,13 +16,11 @@
         "_V_",
         "goog",
         "console",
-
```

Comment:
The line breaks here provided a bit of categorization since we can't use comments in JSON. It's kind of window vars, node vars, and testing vars.